app/novu/emails/novu-onboarding-email.tsx

```diff
@@ -110,9 +72,9 @@ export const NovuWelcomeEmail = ({
             className="mx-auto my-20"
           />
           <Container className="bg-white p-45">
-            {components?.map((item: any, index: any) => {
+            {components?.map((item: EmailComponent, index: number) => {
```

Comment:
```suggestion
            {components?.map((item: EmailComponent, componentIndex: number) => {
```

Reply:
Components is already typed above with `EmailComponent`, there shouldn't be a need to type these parameters out in-line.

```suggestion
            {components?.map((component, componentIndex) => {
```

Reply:
Components is already typed above with `EmailComponent`, we shouldn't need to add in-line typings.

```suggestion
            {components?.map((component, componentIndex) => {
```

Reply:
index renamed, types removed